### DISCUSSION TITLE
What about hard link support ?

### DISCUSSION THREAD

**Discussion by anta999** [elfmz/far2l#1897] [Ideas]
![Screenshot_1](https://github.com/elfmz/far2l/assets/48290431/dd0b98bb-a6a7-4b75-95fc-5ac74549a2d3)
![Screenshot_13](https://github.com/elfmz/far2l/assets/48290431/619e9781-6f12-4096-b37b-081c2815102e)

**Comment by akruphi** [comment]
Now in far2l you can:
* **view** number of hard links in panel mode "File links" (by **LCtrl-9**) in last column
* **create** hard link link or symbolic link by **Alt-F6**

What your use case for hard link additional support in Change Attributes dialog or/and in Files highlighting?

**Reply by anta999** [reply]
Yes, I'm talking about highlighting  and the location of other hard links.

**Comment by atolismesh** [comment]
Finding all the names and locations of hardlink files requires too many resources on Linux, unlike Windows.
You will have to scan the entire file system to find them.
Read this :
https://unix.stackexchange.com/questions/73538/list-all-files-with-the-same-inode-number

**Reply by anta999** [reply]
I've looked all over and it looks like you're right.
